Makefile

```diff
@@ -22,3 +22,15 @@ serve-skore-ui:
 		--host 0.0.0.0 \
 		--port 22140 \
 		--timeout-graceful-shutdown 0
+
+lint:
+	pre-commit run --all-files
+
+test-frontend:
+	cd skore-ui && npm install
+	cd skore-ui && npm run test:unit
+
+test-backend:
+	cd skore && pytest tests
```

Comment:
Could it also run the docs? like `make html`

Reply:
`git clean -dx && make html` to empty the cache?

Reply:
`cd skore/sphinx && git clean -dx && make html`

Reply:
To me building the doc is not a form of testing. Shall we create another target (not included in the test target) for this ?

Reply:
We must be careful with `git clean` because this Makefile can be used during development.
It should not remove WIP, even on documentation.

Reply:
> To me building the doc is not a form of testing. Shall we create another target (not included in the test target) for this ?

Usually, there is a top level `make docs` or equivalent in a lot of package as target

Reply:
> git clean -dx && make html to empty the cache?

Having the cache is actually a feature :). It allows to not have to recompute some stuff. It would make sense to have a `make clean-docs` that remove the `auto-examples` and `generated` folders (I agree with @thomass-dev) regarding the `git` command).

Reply:
That was suppose to be a quick PR to help @MarieS-WiMLDS running tests.
Could we split documentation helper target in another PR ?

Reply:
> Could we split documentation helper target in another PR ?

Absolutely, they are orthogonal issues. For future reference, when it comes to my comments, I might comment because it seems relevant to a particular discussion but they might not required to be address in the PR. I should have probably be explicit that I had not expectations to be in this PR.